### PR TITLE
Feature/rotation

### PR DESCRIPTION
**Pull request by dany-kun**

### CODE REVIEW

File: scissors/src/main/java/com/lyft/android/scissors/CropManager.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    private int rotation = 0;
+
+    public void applyScaleRotation(Matrix matrix) {
+        matrix.postTranslate(-bitmapWidth / 2.0f, -bitmapHeight / 2.0f);
+        matrix.postScale(scale, scale);
+        matrix.postRotate(rotation);
+    }
+
+    public void setRotation(int rotation) {
+        this.rotation = (rotation + 360) % 360;
```
Comment (AlexandreCrettet) [inline scissors/src/main/java/com/lyft/android/scissors/CropManager.java:26]: Why +360 ?

File: scissors/src/main/java/com/lyft/android/scissors/CropManager.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            if (rotation == 90) {
+                x = -x;
+            }
+            if (rotation == 180) {
+                x = -x;
+                y = -y;
+            }
+            if (rotation == 270) {
+                y = -y;
+            }
```
Comment (AlexandreCrettet) [inline scissors/src/main/java/com/lyft/android/scissors/CropManager.java:78]: I don't get these 3 if
Plus use else if

File: scissors/src/main/java/com/lyft/android/scissors/CropManager.java (hunk 1 above)
Comment (dany-kun) [reply scissors/src/main/java/com/lyft/android/scissors/CropManager.java:26]: to be (quite) sure to get something between 0 and 360 (no negative values)

File: scissors/src/main/java/com/lyft/android/scissors/CropManager.java (hunk 2 above)
Comment (dany-kun) [reply scissors/src/main/java/com/lyft/android/scissors/CropManager.java:78]: to be honest, I dont know either...experiments gave me this result, maybe worth to have alook at it later

**Pull request merged**